webpack.config.babel.js

```diff
@@ -28,6 +28,10 @@ const config = {
       },
       {
         test: /\.(glsl|frag|vert)$/, loader: 'glslify-loader', exclude: /node_modules/
+      },
+      {
+        test: /\.wasm$/,
```

Comment:
It seems we don't use `wasm` in this example. If you want to use `wasm` build - [read this](https://gist.github.com/sasha240100/1e0c0894b2dd98623ef68f89d4ef87ab) and get those files. 
You need `ammoloader.js` and `ammo.wasm` files.

```javascript
new WorldModule({
   wasmBuffer: 'ammo.wasm',
   ammo: 'ammoloader.js'
})
```

See [this code](https://github.com/WhitestormJS/physics-module-ammonext/blob/master/src/worker.js#L313).